server/routes/choosePrisonerController.test.ts

```diff
@@ -54,28 +67,35 @@ describe('Choose Prisoner Controller', () => {
       fromLocation: 'Leeds',
       moveType: 'PRISON_TRANSFER',
     },
-  ]
-
-  beforeEach(() => {
-    jest.resetAllMocks()
+  ],
+} as unknown as Map<string, Movement[]>
 
-    controller = new ChoosePrisonerController(incomingMovementsService)
-    incomingMovementsService.getIncomingMovements.mockResolvedValue(incomingMovements)
-  })
+beforeEach(() => {
+  app = appWithAllRoutes({ incomingMovementsService })
+  incomingMovementsService.groupByMoveType.mockResolvedValue(incomingMovementsGroupedByType)
+})
 
-  describe('view', () => {
-    it('should return incoming movements', async () => {
-      await controller.view()(req, res, null)
+afterEach(() => {
+  jest.resetAllMocks()
+})
 
-      expect(res.render).toHaveBeenCalledWith('pages/choosePrisoner.njk', {
-        incomingMovements,
+describe('GET /confirm-arrival/choose-prisoner', () => {
+  it('should contain alphabetically sorted incoming movements grouped by type', () => {
```

Comment:
sorting is an implementation detail that you don't need to expose at this level - the test name suggests that this is a feature of the module under test

Reply:
Updated the test name to 'should render /confirm-arrival/choose-prisoner page'